src/Sample.Extension/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.WebJobs" version="1.2.0-alpha-10306" targetFramework="net45" />
-  <package id="Microsoft.Azure.WebJobs.Core" version="1.2.0-alpha-10306" targetFramework="net45" />
+  <package id="Microsoft.Azure.WebJobs" version="2.0.0-alpha-10324" targetFramework="net45" />
```

Comment:
Wow, no. Good catch. 

Should we file an issue to handle this automatically? We have a script that updates this automatically during the build in our Zumo projects -- we may want to try something similar here.

---
In reply to: [63894366](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63894366) [](ancestors = 63894366)